src/audio/component.c

```diff
@@ -557,3 +557,62 @@ void audio_stream_copy_to_linear(struct audio_stream *source, int ioffset,
 		snk += bytes_copied;
 	}
 }
+
+/** See comp_ops::params */
```

Comment:
I assume this links to the doxygen comment for this call ?

Reply:
I think so, just copy from original place, and paste here.